src/PageUp.CldrPackager/Tools/JsonParsers/Schemas/Main.txt

```diff
@@ -51,10 +51,11 @@
       "type": "object",
       "properties": {
         "_number": 				{ "type": "string" },
-        "_cldrVersion":			{ "type": "string" }
+        "_cldrVersion":			{ "type": "string" },
+        "_pageupVersion":		{ "type": "string" }
```

Comment:
`_pageUpVersion` ?

Reply:
Also add one more tab!

Reply:
source file is _pageupVersion, it was confusing :D

Reply:
Also a classic battle of tabs vs spaces. This file uses a mix of both. In vs the indentation looks fine, but just to be sure i have fixed tabs/spaces in the whole file.